editor/plugins/script_editor_plugin.cpp

```diff
@@ -2182,7 +2182,7 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 
 	script_list = memnew(ItemList);
 	list_split->add_child(script_list);
-	script_list->set_custom_minimum_size(Size2(150 * EDSCALE, 100)); //need to give a bit of limit to avoid it from disappearing
+	script_list->set_custom_minimum_size(Size2(0, 100)); //need to give a bit of limit to avoid it from disappearing
```

Comment:
I think it makes sense, identical comment is in line 2192: `members_overview->set_custom_minimum_size(Size2(0, 100))` and there is 0 in X. I think it's about Y size (you can see on gif it's impossible to squash those panels vertically). Am I wrong?